modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -414,6 +418,16 @@ public void processOpts() {
             additionalProperties.put(POD_AUTHORS, DEFAULT_POD_AUTHORS);
         }
 
+        if (additionalProperties.containsKey(USE_SPM_FILE_STRUCTURE) && (boolean)additionalProperties.get(USE_SPM_FILE_STRUCTURE)) {
+            setSwiftUseApiNamespace(convertPropertyToBooleanAndWriteBack(SWIFT_USE_API_NAMESPACE));
```

Comment:
@aymanbagabas I think maybe what you wanted to do was `setUseSPMFileStructure(convertPropertyToBooleanAndWriteBack(USE_SPM_FILE_STRUCTURE));`?